tern/analyze/docker/run.py

```diff
@@ -181,3 +181,38 @@ def execute_dockerfile(args):  # noqa C901,R0912
     if args.name == 'report':
         if not args.keep_wd:
             report.clean_working_dir()
+
+
+def build_multistage(dockerfile_path, args):
+    """Given a multistage dockerfile, first get the dockerfiles for each
+    stage and then build the dockerfiles.
+    TO DO:
+    1. Add image analyze
+    2. Clean the temp dockerfiles"""
+    dfobj_multi = dockerfile.get_dockerfile_obj(dockerfile_path)
+    file_path_list = dockerfile.get_multistage_image_dockerfiles(dfobj_multi)
+    stage_list = dockerfile.split_multistage_dockerfile_by_stage(dfobj_multi)
+    print(file_path_list)
+    print(stage_list)
+    for idx, dfile in enumerate(file_path_list):
+        args.output_file = '/home/vagrant/output_%d.html' % (idx)
```

Comment:
This is a hard coded path. Also, why the `.html` file extension?

Reply:
This is a draft version, `html` is easy for read.